bitrise.yml

```diff
@@ -183,11 +180,20 @@ workflows:
             go get -u golang.org/x/lint/golint
 
   test-client:
+    envs:
+      - NODE_ENV: "prod"
+    before_run:
+      - create-binaries
     steps:
     - script:
-        title: Karma tests
+        title: Unit tests
         inputs:
-        - content: npm test
+        - content: npm run karma
+    - script:
+        title: E2E tests
+        inputs:
+        - content: npm run e2e-ci
+    - deploy-to-bitrise-io: {}
```

Comment:
**opinion needed**: do we wanna make binaries available as artifacts at every build?